db/versions/0102.yml

```diff
@@ -0,0 +1,80 @@
+version: 102
+description: update pending tasks filter to exclude expired
+methods:
+  get_pending_tasks_by_task_queue_id:
```

Comment:
name stays the same, so will overwrite existing function. 
Input/output stays the same, so there is no change in signature